extensions/wikia/Rail/styles/RailElement.scss

```diff
@@ -0,0 +1,269 @@
+                    font-weight: bold;
+                }
+
+                .chat-total {
+                    background-color: $color-rail-background;
+                    border: 1px solid;
+                    border-radius: 50%;
+                    font-size: 10px;
+                    height: 12px;
+                    margin: -5px 0 0 3px;
+                    padding: 2px;
+                    text-align: center;
+                    width: 12px;
+                }
+            }
+
+            .chat-join {
+                height: 42px;
+
+                button {
+                    background-image: none;
+                    border: 0;
+                    border-radius: 0;
+                    color: $white;
+                    height: 100%;
+                    text-transform: uppercase;
+                }
+
+                button:before {
+                    display: none;
+                }
+            }
+
+            a:hover {
+                text-decoration: none;
+            }
+
+            .avatar {
+                background-position: 50%;
+                border-radius: 50%;
+                height: 30px;
+                margin-right: 14px;
+                width: 30px;
+            }
+
+            .avatar:hover {
+                opacity: 1;
+            }
+
+            .chat-whos-here {
+                border: 0;
+                margin: 20px 0;
+                padding: 0;
+
+                .arrow-left {
+                    padding-left: 0;
+
+                    img {
+                        border-color: transparent;
+                        border-right-color: $color-links;
+                    }
+                }
+
+                .arrow-right {
+                    padding-right: 0;
+
+                    img {
+                        border-color: transparent;
+                        border-left-color: $color-links;
+                    }
+                }
+
+                .arrow-left.disabled, .arrow-right.disabled {
+                    color: mix($color-links, $color-page, 30%);
+                    cursor: pointer;
+                    opacity: .5;
+                }
+
+                .carousel-container {
+                    left: 16px;
+                    width: 260px;
+
+                    li {
+                        width: 32px;
+                    }
+                }
+
+                .UserStatsMenu {
+                    border: $white solid 2px;
+
+                    .actions ul {
+                        margin: 0;
+
+                        li:hover {
+                            background-color: $color-buttons;
+                        }
+
+                        li {
+                            border-bottom: $color-divider solid thin;
+                            cursor: pointer;
+                            height: 44px;
+                            margin: 0;
+                            white-space: nowrap;
+
+                            .label {
```

Comment:
instead of 
```
 li:hover {
   background-color: $color-buttons;
}

 li {
 border-bottom: $color-divider solid thin;
 cursor: pointer;
 height: 44px;
 margin: 0;
 white-space: nowrap;
}
```
you can do 
```
li {
    ...
    &:hover  {
        ...
    }
}